api/envoy/data/dns/v3/dns_table.proto

```diff
@@ -39,8 +39,48 @@ message DnsTable {
     }];
   }
 
-  // This message type is extensible and can contain a list of addresses, clusters or
-  // dictate a different method for resolving the addresses for an endpoint
+  // Specify the service protocol using a numeric or string value
+  message DnsServiceProtocol {
+    oneof protocol_config {
+      option (validate.required) = true;
+
+      uint32 number = 1 [(validate.rules).uint32 = {lt: 255}];
```

Comment:
Please add comments to fields.

Reply:
Certainly.  I will update

Reply:
Add a link to where the numbers can be found. It is from https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml right?